sdk/network/Azure.ResourceManager.Network/src/Generated/ApplicationGateway.cs

```diff
@@ -187,22 +187,23 @@ public virtual ApplicationGatewayDeleteOperation Delete(bool waitForCompletion =
                 throw;
             }
         }
+
         /// <summary> Updates the specified application gateway tags. </summary>
         /// <param name="parameters"> Parameters supplied to update application gateway tags. </param>
         /// <param name="cancellationToken"> The cancellation token to use. </param>
         /// <exception cref="ArgumentNullException"> <paramref name="parameters"/> is null. </exception>
-        public virtual async Task<Response<ApplicationGateway>> UpdateTagsAsync(TagsObject parameters, CancellationToken cancellationToken = default)
+        public async virtual Task<Response<ApplicationGateway>> UpdateAsync(TagsObject parameters, CancellationToken cancellationToken = default)
```

Comment:
Although we have parameter type `TagsObject` to identify, `ApplicationGateway.UpdateAsync` is really confusing...